bedrock/firefox/templates/firefox/new.html

```diff
@@ -6,7 +6,22 @@
 
 {% block page_title_prefix %}{{_('Download Firefox')}} — {% endblock %}
 {% block page_title %}{{_('Free Web Browser')}}{% endblock %}
-{% block page_desc %}{{_('Download Mozilla Firefox, a free Web browser. Firefox is created by a global non-profit dedicated to putting individuals in control online. Get Firefox today!')}}{% endblock %}
+
+{#- This will appear as <meta property="og:title"> which can be used for social share -#}
```

Comment:
https://bugzilla.mozilla.org/show_bug.cgi?id=1140186#c8

"Please don't update the title on /firefox/new/ as that one has already been optimizations for SEO. All other pages, I am fine with. /firefox/new/ is critical to optimize to seo and not brand terms."